template/common-parameter-meta.txt

```diff
@@ -4,7 +4,7 @@ PERMITTED META DESCRIPTIONS:
         modify_memory_gb: "Add to or subtract from dynamic memory allocation. Default memory is determined by the size of the inputs. Specified in GB."
         modify_disk_size_gb: "Add to or subtract from dynamic disk space allocation. Default disk size is determined by the size of the inputs. Specified in GB."
         ncpu: "Number of cores to allocate for task"
-        detect_nproc: "Use all available cores? Recommended for cloud environments. Not recommended for cluster environments."
+        use_all_cores: "Use all cores? Recommended for cloud environments. Not recommended for cluster environments."
```

Comment:
```suggestion
        use_all_cores: "Detect and use all cores? Recommended for cloud environments. Not recommended for cluster environments."
```
I think it's worth clarifying that this detects the number and then uses that.

Reply:
Realizing now that we have this in nearly (?) every task. I'm conflicted about making that change. I wonder if we should hold off and make sure we're happy with the strings in `common-parameter-meta` and then make the update in the Code cleanup PR.